src/validation.cpp

```diff
@@ -5112,7 +5114,9 @@ bool LoadMempool(CTxMemPool& pool)
         file >> mapDeltas;
 
         for (const auto& i : mapDeltas) {
-            pool.PrioritiseTransaction(i.first, i.second);
+            if (!pool.PrioritiseTransaction(i.first, i.second)) {
+                LogPrintf("PrioritiseTransaction(...) failed. Invalid fee delta?\n");
```

Comment:
@practicalswift do you have a PoC for this one too? Just curious what the input looks like.

Reply:
@Crypt-iQ Absolutely, your wish is my command: here you go! :)

Proof of concept:

```
$ git checkout master
$ make -C src/ bitcoind
$ xxd -p -r > mempool-poc-for-second-case-on-line-5115.dat << EOF
0100000000000000010000000000000000000000000000003f0020202020
202020202020202020ff20ff016e2682a8e0af181ae4ffa4ce2fc42b1e3d
8ce88019b194a45cd7c476b88ef1e220202020202020802020ff2020
EOF
$ cp mempool-poc-for-second-case-on-line-5115.dat ~/.bitcoin/regtest/mempool.dat
$ UBSAN_OPTIONS="print_stacktrace=1:halt_on_error=1:report_error_type=1" src/bitcoind -regtest
…
txmempool.cpp:839:15: runtime error: signed integer overflow: -62769881340174304 + -9214329515250016224 cannot be represented in type 'long'
    #0 0x5652ab2203ec in CTxMemPool::PrioritiseTransaction(uint256 const&, long const&) src/txmempool.cpp:839:15
    #1 0x5652ab29e1ad in LoadMempool(CTxMemPool&) src/validation.cpp:5115:18
…
```

:)

Reply:
Add as a test?

Reply:
@promag A fuzz test is added in #19259 ("tests: Add fuzzing harness for LoadMempool(...) and DumpMempool(...)"). Review welcome! :)